bootstrap.php

```diff
@@ -66,6 +66,9 @@
     ->addAlias('Config')
     ->addAlias(Contracts\ConfigurationInterface::class)
 
+    ->rule(\Vanilla\ImageResizer::class)
+    ->addCall('setAlwaysRewriteGif', [false])
```

Comment:
You can set this from the config by using a reference. Frankly, I'm okay with what you did here though.

Reply:
I'm all about only adding new configs as we need them. We have the ability to add one, easily, but I would like to avoid doing so until we know we need it.